tpl/collections/collections.go

```diff
@@ -30,6 +30,11 @@ import (
 	"github.com/spf13/cast"
 )
 
+// This is currently only implemented on Page.
+type grouper interface {
```

Comment:
This interface duplicates the one above. Can we consolidate these two?

Reply:
Not without exporting them.

Reply:
We could do that, of course, but I currently don't see the value.

Reply:
But that value may come sooner rather than later ...

Reply:
OK, I have added a commit where I pulled this out into its own package.